backend/main.py

```
@@ -102,6 +99,7 @@
 from routers import admin_products, admin_subscriptions, admin_credits, admin_analytics, admin_content, admin_settings
 from routers import admin_overview, admin_integrations
 from routers import content, ai, community, session_analytics
+from routers import social_media_marketing_router
```

Ruff (0.12.2)

`routers.social_media_marketing_router` imported but unused

Remove unused import: `routers.social_media_marketing_router`

(F401)